packages/searchable-dropdown/src/provider/controller.ts

```diff
@@ -109,7 +109,8 @@ export class SearchableDropdownController implements ReactiveController {
    * Close dropdown when click oustside host id
    */
   private _handleGlobalClick = (e: MouseEvent): void => {
-    if (e.target && (e.target as HTMLElement).id !== this.#host.id) {
+    const t = e.target as EventTarget;
```

Comment:
I wouldn't typecast the value as it's already `EventTarget | null`.
 I believe `const t = e.target;` or `const { target } = e;` will be sufficient here; and the next line will deal with it being null or not.

Reply:
Done